### ISSUE TITLE
broken links

### ISSUE THREAD

**Issue by barkhaaroraa** [accordproject/techdocs#431]
# Bug Report 🐛

a couple of links are broken on the docs page

## Expected Behavior

they should lead to the documentation
## Current Behavior

<img width="717" alt="image" src="https://github.com/accordproject/techdocs/assets/115429121/84bc8be0-7175-41f8-9d50-aa4bd714f323">
all links under concerto model are broken (ie vocabulary, namespaces, classes, etc)
<img width="950" alt="image" src="https://github.com/accordproject/techdocs/assets/115429121/56f9f954-e7d0-4caf-bc9c-b57117176fd6">

## Possible Solution

the links should lead to the documentation of concerto
<img width="810" alt="image" src="https://github.com/accordproject/techdocs/assets/115429121/59b0d52a-d46b-4c9d-a88a-9b123c3ee570">

## Steps to Reproduce

1. go to https://docs.accordproject.org/docs/accordproject.html
2. click on template guide
3. click on vocabulary, namespaces etc under concerto model

### Desktop
 - OS: [e.g. macOS] windows
 - Browser: [e.g. Chrome, Safari] chrome
 - Version: [e.g. 0.22.15]

## Possible Implementation

if possible i'd like to work on this

**Comment by poojatalele** [comment]
I would like to work on this issue, kindly assign

**Comment by Azhar-2002** [comment]
I would like to work on this issue, Kindly Please assign me.

**Comment by Azhar-2002** [comment]
I fixed the issue of broken line #431 and submitted a pull  request.

**Comment by sanketshevkar** [comment]
I'm not able to see the pull request. I think you've just commited the code to your own fork. You'll have a to create a pull request.

**Comment by Azhar-2002** [comment]
Sorry about that! now i have Created the pull request with issue

**Comment by Azhar-2002** [comment]
Hello Sir/Madam
Last Few Days I am Waiting for the code Review, Please Review my code and
Accept the pull Request.

**Comment by DianaLease** [comment]
@Azhar-2002 The diff in your PR (https://github.com/accordproject/techdocs/pull/445) doesn't seem related to this issue. Can you please explain how it resolves this issue or is there a different PR I should be looking at?

**Comment by Azhar-2002** [comment]
Let me look into it.

**Comment by Azhar-2002** [comment]
You assigned me issue #431 which is working fine. Can You please check it
again. You are checking for issue #455 which is a different issue.

**Comment by sanketshevkar** [comment]
@DianaLease is refering to your PR #445 which is linked to this issue #445 

And I agree with her, I don't think your changes solve the issue. The issue is about updating the docs links and not the vscode config.

**Comment by poojatalele** [comment]
If this issue is still open, I would like to try working on this.

**Comment by sanketshevkar** [comment]
Yes @poojatalele this issue is still open. @Azhar-2002 were you able to fix this?

**Comment by Nandani-sbnk** [comment]
I would also like to work on this issue

**Comment by Anurag1023** [comment]
Is this issue still open?

**Comment by SiddharthBansal007** [comment]
@DianaLease Hi, I would like to help fix this issue, if it isnt resolved yet

**Comment by Karthikmadireddy** [comment]
Hi, I’d like to work on this issue. Please let me know if it’s available.
